src/platform/apollolake/include/platform/platform.h

```diff
@@ -104,7 +104,7 @@ struct sof;
  * the interval of reschedule DMA trace copying in special case like half
  * fullness of local DMA trace buffer
  */
-#define DMA_TRACE_RESCHEDULE_TIME	5000
+#define DMA_TRACE_RESCHEDULE_TIME	500
```

Comment:
So this will mean we send trace every 500us regardless of workload. The intention is that we will send trace less often (say 5ms when quite) and  within 500us when buffer is > 1/2 full. So periodic trace is scheduled every 5ms and when buffer > 1/2 full schedule in 500us.

Reply:
[Question] So periodic trace is scheduled every 5ms and when buffer > 1/2 full schedule in 500us.
[Rander]   The original design is to schedule buffer every 500ms and when buffer > 1/2 full schedule in 5ms.   Now I change the half full schedule time to 500us. Actually for FW of release version, there should be very few trace events. So it is long time to get half full status and schedule in 5ms or 500us, no different. And for debug version, a few trace events maybe enabled by developer, it is possible overflow with 5ms. I also discussed with XiuLi, he even wanted to schedule less than 500us.

Reply:
can we set this to 0, that means we flush when buffer > 1/2 full ASAP?

Reply:
Yes, I test with 0 on APL , no problem. I think 0 is more safe than 500us. It is good point to schedule immediately here, because it is in a trace log function and never interrupt playback or capture.

Reply:
I also test 0 on BDW, no problem, so I decide to change it to 0. 500us is fit for current status, but it maybe decrease to 0 when our topology file become more complex.

Reply:
0 doesn't work on BDW & BYT, but 5us works.  so I decide to change it to 5us.     500us is fit for current status, but it maybe decrease to 5us when our topology file become more complex.